### ISSUE TITLE
[Request] Add alt text to image outputs from /sitelenpona

### ISSUE THREAD

**Issue by stella-sf** [lipu-linku/ilo#41]
At a minimum, the bot would simply add the command input text into the Alt field for all images. I would prefer if it was controlled by an optional boolean argument to the command, so it can be turned off, for example in case you want to have a conversation purely in sitelen pona without anyone "cheating" by reading the alt text. I am unsure whether this argument should default to True or False.

On a higher level of functionality, the bot may remove certain formatting characters from the text to make it easier to read and better for screenreaders as well. This would be a rather complex task especially when considering that different fonts use different systems for their formatting, so this may not be feasible.

Other features that could be added to this include automatically converting UCSUR characters into Latin alphabet text (should be fairly simple with a lookup table), or converting cartouche formatting into readily readable sitelen Lasina names (significantly harder and maybe impossible).

**Comment by gregdan3** [comment]
https://github.com/lipu-linku/ilo/blob/main/src%2Filo%2Fcogs%2Fsitelen%2Fcog.py#L208

We do implement this already, at least the simple version where we copy the user's input. It just doesn't work. IIRC when I examined the command response, the alt text wasn't being sent; pycord bug maybe?
